lib/util.js

```diff
@@ -406,24 +406,27 @@ function inspect(value, opts) {
     maxArrayLength: inspectDefaultOptions.maxArrayLength,
     breakLength: inspectDefaultOptions.breakLength,
     indentationLvl: 0,
-    compact: inspectDefaultOptions.compact
+    compact: inspectDefaultOptions.compact,
+    budget: {}
   };
-  // Legacy...
-  if (arguments.length > 2) {
-    if (arguments[2] !== undefined) {
-      ctx.depth = arguments[2];
-    }
-    if (arguments.length > 3 && arguments[3] !== undefined) {
-      ctx.colors = arguments[3];
+  if (arguments.length > 1) {
+    // Legacy...
```

Comment:
Can you explain this change? Is `opts` a legacy argument now?

Reply:
No, it's just one branch less if no options are used.

Reply:
Oh, I see – the comment just made it seem like it applies to the whole block. Makes sense though :+1: